src/plugins/raven/actions/requestCollaborationNodes.js

```diff
@@ -0,0 +1,28 @@
+/* globals API_ENDPOINT */
+import ObjectSerializer from '@gdbots/pbj/serializers/ObjectSerializer';
+import nodesRecieved from '@triniti/cms/plugins/ncr/actions/nodesRecieved';
+import { actionTypes } from '../constants';
+
+import receiveCollaborationNodes from './receiveCollaborationNodes';
+
+export default (accessToken) => async (dispatch) => {
+  dispatch({
+    type: actionTypes.COLLABORATION_NODES_REQUESTED,
```

Comment:
@nanetteCodes not seeing the COLLABORATION_NODES_REQUESTED used anywhere, can probably get rid of it.  also the "requestBlah" action creator and suffix of _REQUESTED is usually for something that is handled in a saga.  a getCollaborationNodes action creator or just getCollaborations is fine.